js/src/bootstrapMaterialDesign.js

```diff
@@ -140,8 +140,8 @@ const BootstrapMaterialDesign = (($) => {
 
             // add to arrive if present and enabled
             if (document.arrive && this.config.arrive) {
-              $document.arrive(selector, (element) => {  // eslint-disable-line no-loop-func
-                $(element)[jqueryFn](componentConfig)
+              $document.arrive(selector, function(){  // eslint-disable-line no-loop-func
```

Comment:
This change isn't preferred ES2015 syntax;  I am surprised it passed eslint.  `this` is an instance of `BootstrapMaterialDesign` and not the `element` as far as I can tell.

Please test it with the ES2015 syntax and let me know if I am misunderstanding it.